the_list.html

```diff
@@ -28,6 +28,22 @@ <h2>The List</h2>
         </tr>
     </thead>
     <tbody>
+        <tr>
+            <td>2064: Read Only Memories</td>
+            <td>
+                <a href="https://midboss.itch.io/rom">Itch</a>, 
+                <a href="https://store.steampowered.com/app/330820/2064_Read_Only_Memories/">Steam</a>, 
+                <a href="https://www.nintendo.com/games/detail/2064-read-only-memories-integral-switch/">Nintendo Switch</a>, 
+                <a href="https://store.playstation.com/en-us/product/UP0436-CUSA06169_00-READONLYMEMORIES">PS4</a>, 
+                <a href="https://www.microsoft.com/en-us/p/2064-read-only-memories/c53vz4c31g6r?activetab=pivot:overviewtab">Xbox One</a>,
+                <a href="https://play.google.com/store/apps/details?id=com.midboss.rom">Android</a>,
+                <a href="https://apps.apple.com/us/app/read-only-memories-type-m/id1342383410">iOS</a>
+            </td>
+            <td>Visual Novel, Adventure Game, Cyberpunk, Sci-Fi</td>
+            <td>Character Description: Player Character (Choosable pronouns: xe/xem, ze/hir, they/them, she/her, he/him, or a custom pronoun set), TOMCAT (Non-binary, They/Them, Major Character, Confirmed in-game.), Turing (Machine intelligence, they/them, Agender (?), Confirmed in-game)</td>
```

Comment:
Remove "Character Description:".

Also to make easier reading use <div> tags around each character. It will format the entry so that each character is on their own line.